packages/core/stdlib/datetime/main.browse

```diff
@@ -1,14 +1,29 @@
+#*  @name { DateTime }
+# @scope { DateTime utilities }
 import math
 
 import "./native.js"
 
-set SECOND 1000
-set MINUTE $SECOND * 60
-set HOUR $MINUTE * 60
-set DAY $HOUR * 24
-set WEEK $DAY * 7
+#*Number of seconds in a minute
+set SECONDS_PER_MINUTE 60
+#*Number of minutes in an hour
+set MINUTES_PER_HOUR 60
+#*Number of hours in a day
+set HOURS_PER_DAY 24
+#*Number of days in a week
```

Comment:
the previous variables are more useful than these. Most libraries (including pupeteer) uses `ms` and you can compose them.

this module, like the other stdlib ones, are ported from deno

https://deno.land/std@0.69.0/datetime/mod.ts#L5-L10

Reply:
Sorry, ignore this. I didn't see that you moved these variables below 👇